REL/wikipedia.py

```diff
@@ -1,138 +1,155 @@
-import os

```

Comment:
Since this project was originally forked from https://github.com/informagi/REL, which I think was done is dos/windows system, when we ported it to unix environment, we had to change how new lines character returns. When opening a file was worked on widows in Mac/unix system would show `^M` as carriage returns in binary mode. Hence that is why you see the whole file changed when I cleaned these.